qt/python/mantidqt/widgets/plotconfigdialog/curvestabwidget/__init__.py

```diff
@@ -201,7 +201,13 @@ def _get_errorbars_props_from_curve(curve, props):
         """Get a curve's errorbar properties and add to props dict"""
         props['hide_errors'] = getattr(curve, 'hide_errors',
                                        errorbars_hidden(curve))
-        props['errorevery'] = getattr(curve, 'errorevery', 1)
+        # ErrorbarContainer does not have 'errorevery' as an attribute directly
+        # So to get this property take from errorbar lines curve
+        try:
+            barlines = curve[2][0]
+            props['errorevery'] = int(barlines.axes.creation_args[barlines.axes.creation_args.__len__()-1]['errorevery'])
```

Comment:
Can I ask if there is a reason we are using .`__len__` instead of `len()`?

Reply:
I think I did try len() and it didn't work, I can retry again to double check

Reply:
I get an exception 'list' object has no attribute len

Reply:
Does `len(barlines.axes.creation_args)` not work? If it's a `list` it should...

Reply:
okay this was me being forgetful, I was doing `barlines.axes.creation_args.len()` instead of `len(barlines.axes.creation_args)` :facepalm:

Reply:
thanks Martyn